e2e/app/aou-elements/select-component.ts

```diff
@@ -0,0 +1,69 @@
+import {Page} from 'puppeteer';
+
+export default class SelectComponent {
+
+  private readonly page: Page;
+  private readonly name: string;
+
+  constructor(page: Page, name?: string) {
+    this.page = page;
+    this.name = name || undefined;
+  }
+
+  async select(textValue: string) {
+    await this.open(2);
+    const selector = this.componentXpath() + `//li[@class='p-dropdown-item'][normalize-space(.)="${textValue}"]`;
+    const selectValue = await this.page.waitForXPath(selector, { visible: true });
+    await selectValue.click();
+  }
+
+  async getSelectedValue(): Promise<unknown> {
+    const selector = this.componentXpath() + '/label';
+    const displayedValue = await this.page.waitForXPath(selector, { visible: true });
+    const jValue = await (await displayedValue.getProperty('innerText')).jsonValue();
```

Comment:
nit: local var doesn't really do anything, just return?

Reply:
removed. made two lines of code to make clear.
```
const innerText = await displayedValue.getProperty('innerText');
return await innerText.jsonValue();
```